probe/probe.go

```diff
@@ -302,3 +334,362 @@ func probeNode(c *k8s.Client) {
 		fmt.Println("No kubernetes environment found")
 	}
 }
+
+type KubeArmorProbeData struct {
+	OSImage                 string
+	KernelVersion           string
+	KubeletVersion          string
+	ContainerRuntime        string
+	ActiveLSM               string
+	KernelHeaderPresent     bool
+	HostSecurity            bool
+	ContainerSecurity       bool
+	ContainerDefaultPosture tp.DefaultPosture
+	HostDefaultPosture      tp.DefaultPosture
+}
+
+func isKubeArmorRunning(c *k8s.Client) bool {
+	_, err := getKubeArmorDaemonsets(c)
+	return err == nil
+
+}
+
+func getKubeArmorDaemonsets(c *k8s.Client) (bool, error) {
+	var data [][]string
+	// // KubeArmor DaemonSet
+	w, err := c.K8sClientset.AppsV1().DaemonSets("kube-system").Get(context.Background(), "kubearmor", metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+	color.Green("\nFound KubeArmor running in Kubernetes \n\n")
+	_, err = boldWhite.Printf("Daemonset : \n")
+	if err != nil {
+		color.Red(" Error while printing")
+	}
+	desired, ready, available := w.Status.DesiredNumberScheduled, w.Status.NumberReady, w.Status.NumberAvailable
+	if desired != ready && desired != available {
+		data = append(data, []string{"\t kubearmor ", "Desired: " + strconv.Itoa(int(desired)), "Ready: " + strconv.Itoa(int(ready)), "Available: " + strconv.Itoa(int(available))})
```

Comment:
I thought due to the introduction of Tables, we won't need to use tabs anymore?
An empty column might serve the purpose 🤔